src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyServiceTests.cs

```diff
@@ -980,9 +1081,261 @@ await using (var context = InMemoryApplicationDbContext(contentDbContextId))
+                result.AssertNotFound();
+            }
+
+            VerifyAllMocks(cacheService, contentService, imageService, methodologyRepository, publishingService);
+        }
+
+        [Fact]
+        public async Task UpdateMethodology_ApprovingUsingWithReleaseStrategy_ReleaseAlreadyPublished()
+        {
+            var publication = new Publication();
+
+            // Create a release that is already published which the methodology cannot be made dependant on
+            var scheduledWithRelease = new Release
+            {
+                Id = Guid.NewGuid(),
+                Publication = publication,
+                TimePeriodCoverage = CalendarYear,
+                ReleaseName = "2021",
+                Published = DateTime.UtcNow
+            };
+
+            var methodology = new Methodology
+            {
+                PublishingStrategy = Immediately,
+                Status = Draft,
+                MethodologyParent = new MethodologyParent
+                {
+                    Publications = AsList(new PublicationMethodology
+                    {
+                        Owner = true,
+                        Publication = publication
+                    })
+                }
+            };
+
+            var request = new MethodologyUpdateRequest
+            {
+                LatestInternalReleaseNote = "Test approval",
+                PublishingStrategy = WithRelease,
+                WithReleaseId = scheduledWithRelease.Id,
+                Status = Approved,
+                Title = "Title"
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+
+            await using (var context = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await context.Methodologies.AddAsync(methodology);
+                await context.Releases.AddAsync(scheduledWithRelease);
+                await context.SaveChangesAsync();
+            }
+
+            var cacheService = new Mock<ICacheService>(Strict);
+            var contentService = new Mock<IMethodologyContentService>(Strict);
+            var imageService = new Mock<IMethodologyImageService>(Strict);
+            var methodologyRepository = new Mock<IMethodologyRepository>(Strict);
+            var publishingService = new Mock<IPublishingService>(Strict);
+
+            await using (var context = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var service = SetupMethodologyService(contentDbContext: context,
+                    cacheService: cacheService.Object,
+                    methodologyContentService: contentService.Object,
+                    methodologyImageService: imageService.Object,
+                    methodologyRepository: methodologyRepository.Object,
+                    publishingService: publishingService.Object);
+
+                var ex = await Assert.ThrowsAsync<ArgumentException>(() =>
```

Comment:
In the general convention in our codebase, I'd expect this to be a Left result in the form of a  Bad Request, returning some ValidationErrorMessage.  Then we'd be checking here with `result.AssertBadRequest(MethodologyCannotDependOnPublishedRelease)`

Reply:
Not sure why I did it like this. I've swapped it over to an ActionResult so testing for a bad request now like you suggest. Thanks.